### DISCUSSION TITLE
Volume: Volumecontrol disabled

### DISCUSSION THREAD

**Discussion by ghost** [jcorporation/myMPD#582] [Q&A] [answered]
It does not work with hardwear volume.
Thank you very much. 
(In alsamixser, it is recognized as a hardwear mixer and can operate.)

**Comment by jcorporation** [comment, marked as answer]
Can a other mpd client change the volume? Try `mpc volume 50`. 

If this works it is an myMPD issues else a bad mpd configuration.

If it is a myMPD issue, please use the bug template to post valid debug data.

**Reply by ghost** [reply]
sorry! .. I'm using the Collybia OS and I posted it in the wrong place. Try a little more thought. Thank you very much.
**I'm using Google Translate, typographical errors, etc.
Please forgive me.**

**Reply by ghost** [reply]
Thank you for your advice. In the mpd.conf settings, <mixer_control "digital"> was added and the mixer volume slider button appeared and it became operational. Thank you very much.
